strawberryfields/gbs/plot.py

```diff
@@ -0,0 +1,235 @@
+# Copyright 2019 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+r"""
+Plotting and visualization
+==========================
+
+**Module name:** :mod:`strawberryfields.gbs.plot`
+
+.. currentmodule:: strawberryfields.gbs.plot
+
+This module provides functionality for visualizing GBS outputs as graphs and point processes.
+
+.. autosummary::
+    _node_coords
+    edge_coords
+    plot
+    plot_subgraph
+
+Code details
+^^^^^^^^^^^^
+"""
+from typing import Tuple, Optional
+
+import networkx as nx
+import plotly.graph_objects as go
+
+
+def _node_coords(graph: nx.Graph, l: dict) -> Tuple:
+    """ Provides the coordinates for the graph nodes when given an input graph layout.
```

Comment:
I see it as more of a converter, i.e., `l` already has the coordinates, this just separates them out into x and y coordinate arrays that are keys of a dict, for easy passing to some plotly functions.

Reply:
You're right, `l` already has the coordinates. I'll change the docstring to clarify this

Reply:
Yeah, I agree --- this function already takes coordinates as input, so the name/description doesn't seem to align.